fastly/resource_fastly_alert.go

```diff
@@ -4,13 +4,15 @@ import (
 	"context"
 	"errors"
 	"log"
+	"strings"
 
 	gofastly "github.com/fastly/go-fastly/v9/fastly"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 const badAlertSourceServiceIdConfig = "empty `service_id` is only supported for `stats` as a source"
+const managedByTerraform = "Managed by Terraform"
```

Comment:
This should be exported here (or in another file) so that the test can use the same string.

Reply:
This also opens the door to a much larger discussion... are alerts so special that they deserve this sort of tagging, or should we consider doing this more broadly across the provider?